hc/accounts/tests/test_check_token.py

```diff
@@ -22,7 +22,17 @@ def test_it_redirects(self):
         self.assertEqual(self.profile.token, "")
 
     ### Login and test it redirects already logged in
+    def test_it_redirects_when_logged_in(self):
+        self.client.login(username="alice@example.org", password="password")
+
+        r = self.client.post("/accounts/check_token/alice/secret-token/")
+        self.assertRedirects(r, "/checks/")
 
     ### Login with a bad token and check that it redirects
+    def test_it_redirects_after_bad_login(self):
+        url = "/accounts/check_token/alice/invalid-token/"
```

Comment:
I like the way you created a variable for the URL before using it in the post function